src/main/java/com/stripe/net/StripeResponseGetter.java

```diff
@@ -20,4 +21,14 @@
       Class<T> clazz,
       RequestOptions options)
       throws StripeException;
+
+  default InputStream requestStream(
```

Comment:
We do lose some type safety here since it's just a generic InputStream, but I'm not sure what we can really do about that... presumably the few callers that use this will be aware of how they should process the stream. If we wanted, we could make some typed wrapper around InputStream, but then the issue becomes compatibility with functions take operate on streams so I don't think it's worth it.